src/modules/create-market/components/create-market-form-outcome/create-market-form-outcome.jsx

```diff
@@ -41,8 +41,8 @@ export default class CreateMarketOutcome extends Component {
       // marketType: false,
       outcomeFieldCount: CreateMarketOutcome.calculateOutcomeFieldCount(this.props),
       showAddOutcome: CreateMarketOutcome.calculateOutcomeFieldCount(this.props) < 8,
-      scalarMin: createBigNumber(speedomatic.constants.INT256_MIN_VALUE).decimalPlaces(18, BigNumber.ROUND_DOWN),
-      scalarMax: createBigNumber(speedomatic.constants.INT256_MAX_VALUE).decimalPlaces(18, BigNumber.ROUND_DOWN),
+      scalarMin: Number.MIN_SAFE_INTEGER,
```

Comment:
We do not use Javascript Numbers, so these values need to remain as they are defined. The Ethereum Blockchain allows values up to 2^256 but Javascript only goes up to 2^53.